scripts/build_backend.py

```diff
@@ -157,15 +157,15 @@ def custom_build_cmake_clib(
     python_library_dir = win_python_path_lib if IS_WIN else get_config_var("LIBDIR")
     numpy_include = np.get_include()
 
-    if iface == "dpc":
+    if iface in ["dpc", "spmd_dpc"]:

```

Comment:
```suggestion
    if iface in ("dpc", "spmd_dpc"):
```

I think defining list constants is more common.

Reply:
Does it really make sense here?